tests/phpunit/tests/html-api/wpHtmlProcessor.php

```diff
@@ -348,8 +348,15 @@ public static function data_unsupported_special_in_body_tags() {
 	public function test_reports_proper_element_depth_in_body( $html_with_target_element, $depth_at_element ) {
 		$processor = WP_HTML_Processor::create_fragment( $html_with_target_element );
 
-		$this->assertTrue(
-			$processor->next_tag( array( 'class_name' => 'target' ) ),
+		while ( $processor->next_tag() ) {
+			if ( $processor->has_class( 'target' ) ) {
+				break;
```

Comment:
why this change?

Reply:
I thought the other form was broken, causing the test to fail. That's the only reason. I may be mistaken, feel free to revert.

Reply:
oops, this is a known bug I forgot about. we claim to support `class_name` but don't